integrations/Adroll/browser.js

```diff
@@ -98,7 +98,7 @@ class Adroll {
     } else if (message.name && !message.category) {
       pageFullName = `Viewed ${message.name} Page`;
     } else {
-      pageFullName = `Viewed ${message.name} ${message.category} Page`;
+      pageFullName = `Viewed ${message.category} ${message.name} Page`;
```

Comment:
This seems like a breaking change. Can you confirm?
CC @utsabc

Reply:
![Screenshot 2022-05-31 at 7 06 02 PM](https://user-images.githubusercontent.com/25389167/171186613-bdd884cf-b34b-4cc1-aa2f-b275fd9cda60.png)
This is disabled for them its safe to merge